Code/Cleavir/Type-inference/transfer.lisp

```diff
@@ -94,44 +94,76 @@
 		input-bag))))
 
 (defmethod one-successor-transfer
-    ((instruction cleavir-ir:short-float-unbox-instruction) input-bag)
-  (let ((output (first (cleavir-ir:outputs instruction))))
-    (update output 'unboxed-short-float input-bag)))
+    ((instruction cleavir-ir:aref-instruction) input-bag)
+  ;; TODO: update array
+  (let ((index (second (cleavir-ir:inputs instruction)))
+	(output (first (cleavir-ir:outputs instruction)))
```

Comment:
I believe Emacs by default indents using tabs at the beginning of the line, and then spaces to reach the desired column.